scripts/update-drafts.ts

```diff
@@ -137,12 +138,29 @@ async function main() {
       continue;
     }
 
-    const spec_url = feature.data.__compat.spec_url;
-    if (!spec_url) {
-      continue;
+    let parent_url = null;
```

Comment:
I think some types here makes following what comes later a little easier to follow. I'm suggesting this (and some other types, where it helped me follow the logic):

```suggestion
    let parent_url: string | string[] = null;
```